src/main/scala/beam/router/graphhopper/GraphHopperRouteResolver.scala

```diff
@@ -0,0 +1,152 @@
+package beam.router.graphhopper
+
+//import java.io.File
+
+import beam.utils.ProfilingUtils
+import com.conveyal.osmlib.{OSM, OSMEntity}
+import com.conveyal.r5.transit.TransportNetwork
+import com.graphhopper.config.CHProfile
+import com.graphhopper.{GHRequest, GHResponse, GraphHopper}
+//import com.graphhopper.reader.dem.MultiSourceElevationProvider
+import com.graphhopper.config.Profile
+import com.graphhopper.reader.ReaderWay
+import com.graphhopper.routing.ch.{CHPreparationHandler, PrepareContractionHierarchies}
+import com.graphhopper.routing.util.{BikeFlagEncoder, CarFlagEncoder, EncodingManager, FootFlagEncoder}
+import com.graphhopper.routing.weighting.{FastestWeighting, PriorityWeighting, TurnCostProvider}
+import com.graphhopper.storage.{CHConfig, DAType, GHDirectory, GraphHopperStorage}
+import com.graphhopper.util.{PMap, PointList}
+import com.graphhopper.util.shapes.GHPoint
+import com.typesafe.scalalogging.LazyLogging
+import org.matsim.api.core.v01.Coord
+
+import GraphHopperRouteResolver._
+
+class GraphHopperRouteResolver(val ghLocation: String) extends LazyLogging {
```

Comment:
Now [GraphHopperWrapper](https://github.com/LBNL-UCB-STI/beam/blob/develop/src/main/scala/beam/router/graphhopper/GraphHopperWrapper.scala) is on develop, do you think you can easily replace `GraphHopperRouteResolver` by it?